### PR TITLE
Make the check-rate subcommand more visible

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Adds a log line in yellow underneath the output of the `check` command that states running with check-rate will allow you to see non gauge metric types. 

### Motivation

Clear up confusion about the check and check-rate commands

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: cmd/agent/app/check.go [hunk 1]
```diff
@@ -131,6 +131,10 @@ var checkCmd = &cobra.Command{
 			fmt.Println(string(checkStatus))
 		}
 
+		if checkRate == false {
+			color.Yellow("Currently only displaying Gauge metrics. Please run the command with --check-rate to see any other metric types if available.")
```
Comment (masci) [inline cmd/agent/app/check.go]: Can we s/Please/You can ?
"Please" sounds to me like the user did something wrong that need to fix, does it make sense?

Comment (nmuesch) [reply cmd/agent/app/check.go]: Good call, definitely don't want to imply something was done wrong. Updated! 🙇

**Review by masci — APPROVED** [review]

File: cmd/agent/app/check.go [hunk 2]
```diff
@@ -131,6 +131,10 @@ var checkCmd = &cobra.Command{
 			fmt.Println(string(checkStatus))
 		}
 
+		if checkRate == false {
+			color.Yellow("Currently only displaying Gauge metrics. You can run the command with --check-rate to see any other metric types if available.")
```
Comment (olivielpeau) [inline cmd/agent/app/check.go]: @nmuesch: This message is a bit misleading: one check run is enough for other types of check-submitted metrics than just `Gauge`. For instance metrics submitted with `Sender.Count`/`self.count`, `Sender.Histogram`/`self.histogram` are shown without requiring `--check-rate`.

We should probably reword this to mention that the command is "not displaying metrics submitted with `Rate` and `MonotonicCount`", instead of "only displaying Gauge metrics".

Thoughts?

Comment (masci) [reply cmd/agent/app/check.go]: Good point. Best option would be avoid referring to specific metric types, to avoid having to change this string whenever we change something on the aggregator side (unlikely but still).
Something like:
> Check has run only once, if some metrics are missing you can try again with --check-rate to see any other metric if available.

Comment (nmuesch) [reply cmd/agent/app/check.go]: Good call @olivielpeau @masci I've updated the wording to remove specific mention to gauges here - https://github.com/DataDog/datadog-agent/pull/1460
